lib/addons/p5.dom.js

```diff
@@ -901,14 +1013,7 @@
     if (this.elt.currentTime === this.elt.duration) {
       this.elt.currentTime = 0;
     }
-
-    if (this.elt.readyState > 1) {
-      this.elt.play();
-    } else {
-      // in Chrome, playback cannot resume after being stopped and must reload
-      this.elt.load();
-      this.elt.play();
-    }
+    this.elt.play();
```

Comment:
is the comment previously on line 908 no longer true (since which version of chrome)?